src/index.html

```diff
@@ -1,22 +1,63 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900&display=swap" rel="stylesheet" />
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Moyo header</title>
-    <link
-      rel="stylesheet"
-      href="./style.css"
-    />
+    <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <img src="images/logo.png" class="img"
+       alt="logo" />
```

Comment:
This part of the task is incomplete
```suggestion
      <img
        src="images/logo.png"
        class="header__img"
         alt="logo"
       />
```
<img width="825" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/47e4f957-3943-415d-a4a7-cee4d05bae1e">